estate_account/models/estate_property.py

```diff
@@ -0,0 +1,25 @@
+from odoo import fields, models
+
+class EstateProperty(models.Model):
+    _inherit = 'estate.property'
+
+    def action_mark_sold(self):
+        for record in self:
+            lines = [
+                fields.Command.create({
+                    'name': '6% of selling price',
+                    'quantity': 0.06,
+                    'price_unit': record.selling_price,
+                }),
+                fields.Command.create({
+                    'name': 'administrative fees',
+                    'quantity': 1,
+                    'price_unit': 100000,
+                })
+            ]
+            record.env['account.move'].create({
+                    'partner_id': record.buyer_id.id,
+                    'move_type': 'out_invoice',
+                    'invoice_line_ids': lines,
```

Comment:
create a dict of values to create , then create outside of the loop.
Never search / create in loop (if possible of course)